File: js/ubicacion.js
class Ubicacion {
    constructor(id = null, descripcion = null) {
        this._id = id;
        this._descripcion = descripcion;
    }
    get id() {
        return this._id;
    }

    set id(value) {
        this._id = value;
    }

    get descripcion() {
        return this._descripcion;
    }

    set descripcion(value) {
        this._descripcion = value;
    }

}

File: App_Code/Ubicacion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Web;
using System.Xml.Serialization;

/// <summary>
/// Descripción breve de Ubicacion
/// </summary>
namespace es.itsduero.iecscyl
{
    [DataContract]
    public class Ubicacion
    {
        [XmlElement ("UbicacionID")]
        public int ID { get; set; }
        [XmlElement ("UbicacionDescripcion")]
        public string descripcion { get; set; }
    }
}